contrib/epee/src/mlog.cpp

```diff
@@ -204,20 +204,21 @@ void mlog_configure(const std::string &filename_base, bool console, const std::s
           static_assert(std::is_integral<time_t>(), "bad time_t");
           return ta < tb;
         });
-        for (size_t i = 0; i <= found_files.size() - max_log_files; ++i)
+        for (size_t i = 0; i < found_files.size() - max_log_files; ++i)
```

Comment:
If `max_log_files` happens to be 0 for some reason, then accessing the files under the condition of <= will crash.